src/experimental/safe_html_transform/configure.zcml

```diff
@@ -33,4 +33,9 @@
         handler=".setuphandlers.post_install">
     </genericsetup:importStep>
 
+  <include package="plone.app.registry" />
+  <include package="Products.MimetypesRegistry" />
+  <include package="Products.PortalTransforms" />
+  <include package="Products.PortalTransforms" />
```

Comment:
So I should remove all of them as these are  dependencies in metadata.py for default profile ? And Yes PortalTransform was included twice, Will correct that.